appveyor.yml

```diff
@@ -23,5 +30,4 @@ install:
 build: false
 
 test_script:
-  - cd c:\projects\habitat\components\hab
-  - cargo build --verbose
+  - appveyor.bat
```

Comment:
yeah, the problem is the output from cargo via powershell makes me want to stab my eyes.